corehq/apps/integration/models.py

```diff
@@ -13,6 +13,14 @@ class DialerSettings(models.Model):
     dialer_page_subheader = models.CharField(max_length=255)
 
 
+class HmacCalloutSettings(models.Model):
+    domain = models.CharField(max_length=128, unique=True)
```

Comment:
setting `unique` here will also create a `varchar_pattern_ops` index. To avoid the unwanted index you can do this:
https://github.com/dimagi/commcare-hq/blob/b13480f4b8aa2108a52f62aa5128d2fd7ec62d3a/corehq/blobs/models.py#L66-L73